packages/flipper-plugin-react-native-performance/android/gradle.properties

```diff
@@ -0,0 +1,3 @@
+# Supports AndroidX
```

Comment:
IDK, seems like there's a lot of files here that shouldn't be. I think this one should exist in your RN project, not in a lib.